### DISCUSSION TITLE
Need Tab Switch between rows for easy accessibility of plugins

### DISCUSSION THREAD

**Discussion by starkblaze01** [grafana/grafana#38589] [Dashboards, Panels & Visualizations]
Right now we can categorise the plugins in rows and put the plugins into respective rows. But sometimes user doesn't want to look at all the plugins at once and is more focused on a specific category i.e. a particular row. One way is to bring that row to the top, but that's not the ideal case if the number of plugins is more. So, it would be great to have a tab switch like feature for rows to navigate between different categories.

Something similar to the image below:

![image](https://user-images.githubusercontent.com/22374162/130959938-14877938-9b0c-40da-b43b-98b6c5d3f9fa.png)

![image](https://user-images.githubusercontent.com/22374162/130959870-77960d31-cac6-48c4-9af5-d830bf91073f.png)

**Comment by DmitrySidorow** [comment]
Great idea!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
